src/bundle/Resources/views/themes/admin/ui/menu/main_base.html.twig

```diff
@@ -0,0 +1,83 @@
+{% extends '@KnpMenu/menu.html.twig' %}
+{% import 'knp_menu.html.twig' as knp_menu %}
+
+{% block popupItem %}
+    {% if item.displayed %}
+        {% set node_attributes = {
+            'class': 'ibexa-popup-menu__item',
+        } %}
+
+        <li {{ knp_menu.attributes(node_attributes) }}>
+            {%- if item.uri is not empty and (not matcher.isCurrent(item) or options.currentAsLink) %}
+                {{ block('popupLinkElement') }}
+            {%- else %}
+                {{ block('popupSpanElement') }}
+            {%- endif %}
+        </li>
+    {% endif %}
+{% endblock %}
+
+{% block popupLinkElement %}
+    {% set classes = [
+        'ibexa-popup-menu__item-action',
+        ((matcher.isCurrent(item) ? 'ibexa-popup-menu__item-action--current')),
```

Comment:
does it work correctly? Doesn't it add null element to array?

Reply:
> does it work correctly? Doesn't it add null element to array?

yes, the only inconvenience is empty string on the end after `join`